api/src/test/java/org/pmiops/workbench/api/OfflineAuditControllerTest.java

```diff
@@ -41,8 +39,6 @@
 
 @RunWith(SpringRunner.class)
```

Comment:
If we need these 4 annotations for all our tests, I wonder if we can't either make a new annotation that combines them or a parent class that sets these for us. Not for this PR, though.

Reply:
Agreed.